packages/mkocaml/mkocaml.1.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+synopsis: "Tool to generate OCaml projects"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/mkocaml"
+bug-reports: "https://github.com/chrisnevers/mkocaml/issues"
+depends: [
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11.0"}
```
According to your dune-project file you need at least dune 1.11.
Also dune cannot be tagged as build dependency: https://github.com/ocaml/opam-repository/issues/14185